docs/source/developer_guide/tutorial.rst

```diff
@@ -150,3 +150,20 @@ Running these tests requires an additional flag to be passed to ``pytest``::
 Alternatively, using ``tox``::
 
     tox -e extra-mlips
+
+Adding a new Observable
+=======================
+
+Additional built-in observable quantities may be added for use within the ``janus_core.helpers.correlator.Correlation``s. These should conform to the ``__call__`` signature of ``janus_core.helpers.janus_types.Observable``. For a user this can be accomplished by writing a function, or class also implementing a commensurate ``__call__``.
```

Comment:
```suggestion
Additional built-in observable quantities may be added for use by the ``janus_core.helpers.correlator.Correlation`` class. These should conform to the ``__call__`` signature of ``janus_core.helpers.janus_types.Observable``. For a user this can be accomplished by writing a function, or class also implementing a commensurate ``__call__``.
```